estuary/utils/story.py

```diff
@@ -0,0 +1,468 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import abc
+from neomodel import db
+
+from estuary.error import ValidationError
+from estuary.models.koji import ContainerKojiBuild, KojiBuild, ModuleKojiBuild
+from estuary.models.bugzilla import BugzillaBug
+from estuary.models.distgit import DistGitCommit
+from estuary.models.errata import Advisory, ContainerAdvisory
+from estuary.models.freshmaker import FreshmakerEvent
+
+
+class BaseStoryManager(object):
+    """A class containing utility methods to create a story for an artifact."""
+
+    @staticmethod
+    def get_story_manager(item, item_id, config, limit=False):
+        """
+        Select which story flow to follow.
+
+        :param node item: a Neo4j node whose story is requested by the user
+        :param int node_id: the internal Neo4j ID of the node
+        :kwarg bool limit: specifies if LIMIT keyword should be added to the created cypher query
+        :param flask.config.Config config: flask config
+        :return: stories of a node in forward and backward direction
+        :rtype: list
+        """
+        for class_name in config.get('STORY_MANAGER_SEQUENCE'):
+            story_manager = eval(class_name)()
```

Comment:
I hate eval because it runs arbitrary code. How about something like:
```python
story_manager_cls = globals().get(class_name)
if not story_manager_cls:
     # raise exception here
story_manager = story_manager_cls()
```